run/jobs/main.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"strconv"
+	"time"
+)
+
+func getEnvVars() (string, string, string, string) {
+	// Retrieve Job-defined env vars
+	var TASK_NUM = os.Getenv("TASK_NUM")
```

Comment:
Unexported constants should be lowerCamelCase.
https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps